pyccl/core.py

```diff
@@ -510,14 +485,6 @@ def _build_config(
                              % (emulator_neutrinos,
                                 emulator_neutrinos_types.keys()))
 
-        # Check for valid transfer fn/matter power spectrum combination
-        if (matter_power_spectrum_types[matter_power_spectrum]
```

Comment:
I think we are better off without it.

Reply:
I'm a bit worried about not checking for consistency. At least we should have an explanation in the documentation of which cases are consistent with each other and which are not.

Reply:
None of the cases are consistent. 

CLASS is using a modified version of HALOFIT with fairly arbitrary changes to accommodate neutrinos (not published) and w0-wa (not tested with neutrinos). The emulator does not provide a linear power spectrum and so we cannot promise CLASS used with it will match.